modules/twinklewarn.js

```diff
@@ -459,33 +589,27 @@ Twinkle.warn.messages = {
 				label: "Page moves against naming conventions or consensus",
 				summary: "Caution: Page moves against naming conventions or consensus"
 			},
-			"uw-tpv2": {
-				label: "Refactoring others' talk page comments",
-				summary: "Caution: Refactoring others' talk page comments"
-			},
-			"uw-upload2": {
-				label: "Uploading unencyclopedic images",
-				summary: "Caution: Uploading unencyclopedic images"
-			}
-		}/*,
-		"To be removed from Twinkle": {
 			"uw-redirect2": {
 				label: "Creating malicious redirects",
 				summary: "Caution: Creating malicious redirects"
 			},
-			"uw-ics2": {
-				label: "Uploading files missing copyright status",
-				summary: "Caution: Uploading files missing copyright status"
+			"uw-speedy2": {
+				label: "Removing speedy deletion tags",
+				summary: "Caution: Removing speedy deletion tags"
+			},
+			"uw-tpv2": {
+				label: "Refactoring others' talk page comments",
+				summary: "Caution: Refactoring others' talk page comments"
 			}
-		}*/
+		}
 	},
 
 
 	level3: {
 		"Common warnings": {
 			"uw-vandalism3": {
-				label: "Vandalism",
-				summary: "Warning: Vandalism"
+				label: "Blatant vandalism",
+				summary: "Warning: Blatant vandalism"
```

Comment:
"Blatant" is used as information and clarification in the Twinkle interface. It does not belong into the public edit summary.

Reply:
Feel free to merge unmodified, I'll fix this shortly afterwards. It won't hurt and seems to be the easiest and fastest solution. 😄